1.0/docs/devguide/styling.md

```diff
@@ -8,11 +8,27 @@ subtitle: Developer guide
 
 {% include toc.html %}
 
-
 Polymer uses [Shadow DOM styling
 rules](http://www.html5rocks.com/en/tutorials/webcomponents/shadowdom-201/) for
-providing scoped styling of the element's local DOM.  Scoped styles should be
-provided via `<style>` tags placed inside the element's local DOM `<template>`.
+providing scoped styling of the element's local DOM. This allows elements to
+create style rules that will only apply to the element's light DOM or local
+DOM, as well as advanced theming with CSS custom properties and CSS mixins.
+
+Scoped styles should be provided via `<style>` tags placed inside the element's
+local DOM `<template>`.
+
+To place styles outside of the element, or share styles between elements, you
+can create a [style module](#style-modules).
+
+**NOTE: I think a nice introduction, without code, works a lot better. I also added a brief explanation.**
```

Comment:
Yeah, I agree. 

(Longer-term plan, BTW, is to separate conceptual docs ("How shadow DOM scoping works") from task-oriented docs ("Style a local DOM element"). But working on that ATM for data binding; styling is probably a ways off.